opendevin/events/observation/browse.py

```diff
@@ -5,35 +5,65 @@
 from .observation import Observation
 
 
+def _remove_fields(obj, fields: set[str]):
```

Comment:
might make sense to put `_remove_fields` in the `events` folder so other actions/observations can use it